config/rbac/service_account.yaml

```diff
@@ -1,5 +1,7 @@
 apiVersion: v1
 kind: ServiceAccount
 metadata:
+  labels:
+    azure.workload.identity/use: "true" 
```

Comment:
This can be removed already as you're using `v0.15.0` and hopefully updating to `v1.0.0` as part of this PR